tests/utils.ts

```diff
@@ -1,10 +1,9 @@
 import * as fs from "fs";
 import * as crypto from "crypto";
 import * as http from "http";
-import { ConsoleMessage, Page } from 'puppeteer';
 
 export class Utils {
```

Comment:
it's easy to misuse testing Utils

Reply:
updated all the services and utils names to prepend the test Prefix, thank you for the review